tests/platform_helper/providers/test_ecs.py

```diff
@@ -0,0 +1,206 @@
+import boto3
+import pytest
+from moto import mock_aws
+
+from dbt_platform_helper.exceptions import NoClusterError
+from dbt_platform_helper.providers.ecs import addon_client_is_running
+from dbt_platform_helper.providers.ecs import check_if_ecs_exec_is_available
+from dbt_platform_helper.providers.ecs import get_cluster_arn
+from dbt_platform_helper.providers.ecs import get_or_create_task_name
+from tests.platform_helper.conftest import mock_parameter_name
+from tests.platform_helper.conftest import mock_task_name
+
+env = "development"
+
+
+@mock_aws
+def test_get_cluster_arn(mocked_cluster, mock_application):
+    """Test that, given app and environment strings, get_cluster_arn returns the
+    arn of a cluster tagged with these strings."""
+
+    assert (
+        get_cluster_arn(
+            mock_application.environments[env].session.client("ecs"), mock_application.name, env
```

Comment:
I think it would be clearer if the `env` was declared in line in each test.
```suggestion
            mock_application.environments[env].session.client("ecs"), mock_application.name, 'dev'
```

Reply:
I can update, understand the reasoning. I just like extracting common variables